src/tests/Regressions/coreclr/GitHub_87879/test87879.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- Needed for mechanical merging of all remaining tests, this particular project may not actually need process isolation -->
```

Comment:
This comment should not be included in the new tests. Also, this test should not require process isolation.

Reply:
If the test fails, it will crash the process and thus kill all the other tests. OOME unit tests are a pretty bad experience overall, especially when not isolated, in my experience.

Reply:
ok